### DISCUSSION TITLE
Plans to support parsing of CAS generated by CDSL?

### DISCUSSION THREAD

**Discussion by nitinkgp23** [codereverser/casparser#71] [Ideas]
Hi, checking whether there are plans to support parsing of CAS generated by CDSL, as it is much richer in info (contains all stock holdings too alongwith mutual funds).

If no near-term plans, has there been any effort in this direction? I could pitch in, or start from somewhere if some work has already been done.

**Comment by codereverser** [comment]
Yeah. it is a much desired feature. Unfortunately I don't have access to enough CSDL and NSDL statements that covers all scenarios (stocks, mutualfunds in both demat and SoA form) . PRs are welcome though.
